### PR TITLE
Drivers trading

### PR DESCRIPTION
**Pull request by danilupion**

### CODE REVIEW

File: src/styles.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import "~@angular/material/prebuilt-themes/pink-bluegrey.css";
+@import "theme.scss";
```
Comment (vicmator) [inline src/styles.scss:2]: (opt) I think that would be good idea to put here a common font-family like mat-table (Roboto, "Helvetica Neue", sans-serif)

BEFORE
captura de pantalla 2018-04-01 a las 14 38 34

AFTER
captura de pantalla 2018-04-01 a las 14 38 16

File: src/app/components/header/header.component.html [hunk 2]
```diff
@@ -25,7 +25,7 @@
     </button>
   </mat-menu>
 
-  <button mat-button [matMenuTriggerFor]="championshipsMenu">
+  <button *ngIf="championshipsService.items.length > 1" mat-raised-button color="primary" [matMenuTriggerFor]="championshipsMenu">
     <span>{{ championshipsService.selected }}</span>
```
Comment (vicmator) [inline src/app/components/header/header.component.html]: (opt) what do you think about text inside a button instead to put a span?

File: src/app/components/header/userMenu/userMenu.component.html [hunk 3]
```diff
@@ -11,6 +11,6 @@
   </button>
 </mat-menu>
 
-<button mat-button [matMenuTriggerFor]="userMenu">
```
Comment (vicmator) [inline src/app/components/header/userMenu/userMenu.component.html:14]: The same as I said before about span inside a button

File: src/app/pages/main/drivers/drivers.page.html [hunk 4]
```diff
@@ -2,24 +2,28 @@
...
 
       <ng-container matColumnDef="Driver">
         <mat-header-cell *matHeaderCellDef> Piloto </mat-header-cell>
         <mat-cell *matCellDef="let driver">
           <img class="flagImg" src="../../../../assets/icons/flags/{{driver.countryCode}}.png" alt="flag">
-          <rm-driver [driverCode]="driver.code"></rm-driver>
+          <rm-driver [driver]="driver"></rm-driver>
         </mat-cell>
       </ng-container>
+      <ng-container matColumnDef="Initial Price">
```
Comment (vicmator) [inline src/app/pages/main/drivers/drivers.page.html:16]: (only doubt) Why we need to know what is the initial value and the actual value?

**Review by vicmator — APPROVED** [review]

File: src/styles.scss (hunk 1 above)
Comment (danilupion) [reply src/styles.scss:2]: Done

File: src/app/components/header/header.component.html (hunk 2 above)
Comment (danilupion) [reply src/app/components/header/header.component.html]: Makes sense, that span is redundant

File: src/app/components/header/userMenu/userMenu.component.html (hunk 3 above)
Comment (danilupion) [reply src/app/components/header/userMenu/userMenu.component.html:14]: Got rid of this and several others

File: src/app/pages/main/drivers/drivers.page.html (hunk 4 above)
Comment (danilupion) [reply src/app/pages/main/drivers/drivers.page.html:16]: I thought it added value... if you disagree we can discuss

**Pull request merged**